validator/client/beacon-api/test_handlers.go

```diff
@@ -0,0 +1,73 @@
+//go:build use_beacon_api
+// +build use_beacon_api
+
+package beacon_api
```

Comment:
If this file is only used for testing, name the file aptly with the `_test.go` suffix so that it is not linked into the production binary.